themes/shezaan/shezaan.theme.bash

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+SCM_THEME_PROMPT_DIRTY=" ${red}✗"
+SCM_THEME_PROMPT_CLEAN=" ${bold_green}✓"
+SCM_THEME_PROMPT_PREFIX=" |"
+SCM_THEME_PROMPT_SUFFIX="${green}|"
+
+GIT_THEME_PROMPT_DIRTY=" ${red}✗"
+GIT_THEME_PROMPT_CLEAN=" ${bold_green}✓"
+GIT_THEME_PROMPT_PREFIX=" ${green}|"
+GIT_THEME_PROMPT_SUFFIX="${green}|"
+
+RVM_THEME_PROMPT_PREFIX="|"
+RVM_THEME_PROMPT_SUFFIX="|"
+
+myip=`$BASH_IT/lib/get_ip.py`
```

Comment:
See comment above.

Either

* Inline this if you only want to use this in your theme, and you know that you always have Python available, e.g. using `python -c "import socket; print(socket.gethostbyname(socket.gethostname()));"`; or
* Create a generic, Bash-only solution and make it available as part of the `base` plugin; or
* Create a generic, Bash-only solution and make it available as part of the `base.theme.bash` file (https://github.com/Bash-it/bash-it/blob/master/themes%2Fbase.theme.bash). This way, other themes could use the same functionality.